source/kraken/realtime.cpp

```diff
@@ -646,6 +646,12 @@ void handle_realtime(const std::string& id,
         }
         return;
     }
+    if (meta_vj_exists && is_added_trip(trip_update)) {
+        LOG4CPLUS_WARN(log, "Cannot add twice a new trip"
```

Comment:
```suggestion
        LOG4CPLUS_WARN(log, "Cannot add twice a new trip, rejecting this trip"
```
always be clear on what action as be taken in log

Reply:
better : `cannot add a new trip with id that already exists`